src/platformIcon.tsx

```diff
@@ -48,6 +48,7 @@ export const PLATFORM_TO_ICON = {
   "go-martini": "martini",
   "go-negroni": "go",
   godot: "godot",
+  hydrogen: "hydrogen",
```

Comment:
hydrogen is a JS framework, so for consistency I suggest we go with:

```suggestion
  javascript-hydrogen: "hydrogen",
```